sphinx_shiguredo_theme/layout.html

```diff
@@ -62,7 +62,7 @@
     <aside class="left-sidenavi">
       <div class="left-sidenavi-container">
         <div class="left-sidenavi-sticky">
-          <div class="left-sidenavi-header{{' with-attention-message' if theme_attention_message}}">
+          <div class="left-sidenavi-header{{' with-attention-message' if theme_attention_message}}"{% if theme_left_sidenavi_header_bgcolor %} style="background-color: {{ theme_left_sidenavi_header_bgcolor }}"{% endif %}>
```

Comment:
背景色の変更以上のことをやる可能性がある場合、可能な限り設定の柔軟性を高めるために `style="..."` の部分を任意に指定できるようにしておく案も考えられます。

```suggestion
          <div class="left-sidenavi-header{{' with-attention-message' if theme_attention_message}}"{% if theme_left_sidenavi_header_style %} style="{{ theme_left_sidenavi_header_style }}"{% endif %}>
```